OutOfSchool/OutOfSchool.WebApi/Services/SubsubcategoryService.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Localization;
+using OutOfSchool.Services.Models;
+using OutOfSchool.Services.Repository;
+using OutOfSchool.WebApi.Extensions;
+using OutOfSchool.WebApi.Models;
+using Serilog;
+
+namespace OutOfSchool.WebApi.Services
+{
+    /// <summary>
+    /// Service with business logic for SubsubcategoryController.
```

Comment:
maybe just 'Subsubcategory Service' ?
or put some high level details about what this service does
but no reference to controller, it is not needed